WMS ID 11284 New Analytics Sprints

NagwangGyamtso opened the pull request:
# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved


# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [ ] All filenames are lower case (including folders, images, files, etc.)
- [ ] Filenames are descriptive
- [ ] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [ ] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [ ] Image references in markdown contain an alternate text

arabellayao reviewed (changes requested):
Hi!
1. Please change the filename to all lower case: ![Screen Shot 2022-10-27 at 10 25 16 AM](https://user-images.githubusercontent.com/48001060/198357714-5d610c88-1518-4b43-94ff-8d0d964f9dbc.png)
2. I don't see Krishna Prasad Kotti and Lucian Dinescu names listed in the workshop team - you may want to add them in.

NagwangGyamtso commented:
Hi,

I removed the folder since I’m still drafting it so will upload the completed version later. I added Lucian just now but for Krishna, he won’t be creating any commits – I’m doing the GitHub work on his behalf so it’s not necessary to add him.

Thanks!
Nagwang Gyamtso

[Oracle]<http://www.oracle.com/>
Nagwang Gyamtso
Product Manager, Product Strategy, Analytics
LinkedIn: Linkedin.com/in/nagwanggyamtso/<https://www.linkedin.com/in/nagwanggyamtso/>
Mobile: +1.646.637.3997
Oracle Analytics

From: Arabella Yao ***@***.***>
Date: Thursday, October 27, 2022 at 1:29 PM
To: oracle-livelabs/sprints ***@***.***>
Cc: Nagwang Gyamtso ***@***.***>, Author ***@***.***>
Subject: [External] : Re: [oracle-livelabs/sprints] WMS ID 11284 New Analytics Sprints (PR #86)

@arabellayao requested changes on this pull request.

Hi!

  1.  Please change the filename to all lower case: [Screen Shot 2022-10-27 at 10 25 16 AM] <https://urldefense.com/v3/__https:/user-images.githubusercontent.com/48001060/198357714-5d610c88-1518-4b43-94ff-8d0d964f9dbc.png__;!!ACWV5N9M2RV99hQ!PuA3mUQc7i7UiSjio61fLszyDmfMci23aAsz8bcSWNDWR0dOnNFVV8syZesfdikCMrdIqeVW-bbJgwTdkqbGz2XWfdN_rA$>
  2.  I don't see Krishna Prasad Kotti and Lucian Dinescu names listed in the workshop team - you may want to add them in.

—
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/oracle-livelabs/sprints/pull/86*pullrequestreview-1158718114__;Iw!!ACWV5N9M2RV99hQ!PuA3mUQc7i7UiSjio61fLszyDmfMci23aAsz8bcSWNDWR0dOnNFVV8syZesfdikCMrdIqeVW-bbJgwTdkqbGz2V2Z68AjQ$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/AKJEPOX3DEBQMDWNQ7IXGZDWFK3VXANCNFSM6AAAAAARQFDSHQ__;!!ACWV5N9M2RV99hQ!PuA3mUQc7i7UiSjio61fLszyDmfMci23aAsz8bcSWNDWR0dOnNFVV8syZesfdikCMrdIqeVW-bbJgwTdkqbGz2War7o7nQ$>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
